python/cudf/cudf/_lib/unaryops.pyx

```diff
@@ -161,3 +161,30 @@ def nans_to_nulls(py_col):
         )
 
     return mask
+
+
+def nats_to_nulls(py_col):
+    from cudf.core.column import as_column
+
+    py_col = as_column(py_col)
+
+    cdef gdf_column* c_col = column_view_from_column(py_col)
```

Comment:
You're leaking this column

Reply:
Don't we have anything to free scalar ?

Reply:
The `column_view_from_column` function allocates a new `gdf_column` object that is expected to be freed. You aren't leaking device memory, apologies if that was unclear.

Reply:
I understood that one. 

But as I had to modify the functionality to replace value with null, I had to introduce gdf_scalar variable for `value`, and I couldn't find an API to free that one in Cython.

Reply:
I was thinking that gdf_scalar had one more API to free the memory, but I can just use free/del.